.github/workflows/build-test-package.yml

```diff
@@ -152,8 +165,57 @@ jobs:
 
     - name: 'Build 🐍 Python 📦 package'
       run: |
+        pwd
+        ls -l
         export ITK_PACKAGE_VERSION=${{ matrix.itk-python-git-tag }}
+        git clone https://github.com/InsightSoftwareConsortium/ITKBSplineGradient.git
+        cp ./dockcross-manylinux-download-cache-and-build-module-wheels.sh ./ITKBSplineGradient
+        pushd ITKBSplineGradient
+        git checkout ${{env.ITKBSplineGradient-git-tag}}
+        ./dockcross-manylinux-download-cache-and-build-module-wheels.sh cp${{ matrix.python-version }}
+        popd
+        cp ITKBSplineGradient/include/* include/
+        mv ./ITKBSplineGradient/ITKPythonPackage .
+        mv ./ITKBSplineGradient/tools .
+        git clone https://github.com/InsightSoftwareConsortium/ITKHigherOrderAccurateGradient.git
+        cp ./dockcross-manylinux-download-cache-and-build-module-wheels.sh ./ITKHigherOrderAccurateGradient
+        pushd ITKHigherOrderAccurateGradient
+        git checkout ${{env.ITKHigherOrderAccurateGradient-git-tag}}
+        mv ../ITKBSplineGradient/doxygen-1.8.11.linux.bin.tar.gz .
+        mv ../ITKBSplineGradient/ITKPythonBuilds-linux.tar.zst .
+        ./dockcross-manylinux-download-cache-and-build-module-wheels.sh cp${{ matrix.python-version }}
```

Comment:
If we download the cache again, we are overwriting the previous build. Instead of calling `./dockcross-manylinux-download-cache-and-build-module-wheels.sh`, call `dockcross-manylinux-build-module-wheels.sh`:

https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/b0a8fd3d1a221eefd4c07a0bd2093a38b0cfdd30/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh#L56